community-membership.md

```diff
@@ -296,5 +297,69 @@ with the project. When a contributor returns to being more active, they may be
 promoted back to their previous position at the discretion of the current
 maintainers following the process defined above.
 
+## Specification Sponsor
+
+Specification sponsors are trusted collaborators of the technical committee, and
+work to review, approve, and sponsor [opentelemetry-specification][] issues and
+PRs.
+
+Members are defined
+in [Community Members -> Specification and Protos](./community-members.md#specifications-and-proto)
+and are members of the `spec-sponsors` team (TODO: add link to github team when
+available). Specification sponsors SHOULD list
+their [areas of interest](./areas-of-interest.md).
+
+### Requirements
+
+Specification sponsors are trusted to drive specification work which can be
+complex and which highly impactful to downstream language SIGs and to the
+OpenTelemetry project as a whole. Approvals from specification sponsors count
+towards the approvals required to merge specification PRs. Therefore, there is a
+high bar to becoming a specification sponsor.
+
+- Contributor to OpenTelemetry project for at least 1 year
+- Has been the reviewer or author (through the sponsorship of another
+  specification sponsor) of at least 10 substantial PRs to specification or
+  related repositories
+- Demonstrated sound technical judgement, and a deep understanding of the
+  specification and the OpenTelemetry project
+- Nominated by technical committee member (
+  see [becoming a specification sponsor](#becoming-a-specification-sponsor))
+
+### Responsibilities and privileges
+
+- Review, respond to, and approve issues / PRs when tagged or when within areas
+  of interest (inactivity for more than 1 month when tagged may result in
+  suspension until active again)
+- Sponsor issues, taking responsibility to be point on requirement gathering,
```

Comment:
> Sponsor issues

Is this a responsibility, or a privilege? Are sponsors _expected_ to sponsor issues within their areas of expertise, or are they allowed to be sponsors for issues in areas they care about?

Reply:
Privilege, mostly.

> Are sponsors expected to sponsor issues within their areas of expertise

I'm not sure. I lean towards yes, but its hard to compel people to do things in OSS. Is the TC expected to drive popular spec issues? I think they ought to because solving popular issues helps the project by helping users. But there's no enforcement mechanism for this. Similarly, I think sponsors ought to be cognizant of the popular issues for their areas of interest / expertise, but its hard to imagine compelling them to take responsibility for those.

> or are they allowed to be sponsors for issues in areas they care about

They should be allowed to sponsor any issue, but will naturally gravitate towards their areas of interest / expertise. Limiting to only their areas of interest / expertise would be too rigid.